src/main/java/org/jenkinsci/plugins/ewm/definitions/DiskPool.java

```diff
@@ -57,6 +60,16 @@ public String getDescription() {
         return description;
     }
 
+    @CheckForNull
+    public String getWorkspaceTemplate() {
+        return workspaceTemplate;
+    }
+
+    @DataBoundSetter
+    public void setWorkspaceTemplate(String workspaceTemplate) {
```

Comment:
Maybe makes sense to extend constructor. We're in the alpha stage, so no problems with binary compatibility